app/electron/main.js

```diff
@@ -16,25 +19,59 @@ const Store = require("secure-electron-store").default;
 const ContextMenu = require("secure-electron-context-menu").default;
 const path = require("path");
 const fs = require("fs");
+const log = require("electron-log");
+
 const Protocol = require("./protocol");
 const MenuBuilder = require("./menu");
 
 const isDev = process.env.NODE_ENV === "development";
 const port = 40992; // Hardcoded; needs to match webpack.development.js and package.json
 const selfHost = `http://localhost:${port}`;
 
+// Setup file logging
+
+log.transports.file.level = "info";
+log.transports.file.file = `${__dirname}logger.log`;
```

Comment:
**This is deprecated in docs** : https://github.com/megahertz/electron-log/blob/master/docs/file.md#file-string-deprecated, do not use this.